vitest.setup.js

```diff
@@ -1,9 +1,14 @@
-import { expect, afterEach } from "vitest";
+import { expect, afterAll, afterEach, vi } from "vitest";
 import { cleanup } from "@testing-library/react";
 import * as matchers from "@testing-library/jest-dom/matchers";
 
 expect.extend(matchers);
+window.scrollTo = vi.fn();
```

Comment:
prevents an error in the console when running tests. purely just for reducing noise.